sysctl/sp-rich-core.conf

```diff
@@ -7,3 +7,7 @@ fs.suid_dumpable = 2
 
 # Set the core pattern so that cores are piped into our rich-core-dumper script.
 kernel.core_pattern=|/usr/sbin/rich-core-dumper --pid=%p --signal=%s --name=%e
+
+# Set core pipe limit to prevent kernel from clearing the process info from
+# /proc/<pid> before the handler has finished
+kernel.core_pipe_limit=4
```

Comment:
I think we should follow systemd here and set the limit to 16 as maximum process that can be spawned for coredumps.
This way if a coredump takes no longer to create or more processes crash they can still be dumped.

https://github.com/systemd/systemd/issues/17301

Reply:
Let's start with this. To me 16 would sound quite a lot in mobile device context.